src/core/include/units/math.h

```diff
@@ -147,4 +147,43 @@ template<Quantity Q>
   return Q(std::numeric_limits<typename Q::rep>::epsilon());
 }
 
+/**
+ * @brief Returns the greatest quantity that is representable in Unit To that is less or equal to q.
```

Comment:
This reads like `std::chrono`'s wrong specification. Check out how `std::floor` is specified here: https://en.cppreference.com/w/cpp/numeric/math/floor.